__tests__/GithubUtils-test.js

```diff
@@ -85,6 +109,58 @@ describe('GithubUtils', () => {
         });
     });
 
+    describe('updateStagingDeployCash', () => {
+        test('successfully updates issue', () => {
+            const issueBefore = {
+                url: 'https://api.github.com/repos/Expensify/Expensify/issues/29',
+                title: 'Test StagingDeployCash',
+                labels: [
+                    {
+                        id: 2783847782,
+                        node_id: 'MDU6TGFiZWwyNzgzODQ3Nzgy',
+                        url: 'https://api.github.com/repos/Expensify/Expensify/labels/StagingDeployCash',
+                        name: 'StagingDeployCash',
+                        color: '6FC269',
+                        default: false,
+                        description: ''
+                    }
+                ],
+                // eslint-disable-next-line max-len
+                body: '**Release Version:** `1.0.1-472`\r\n**Compare Changes:** https://github.com/Expensify/Expensify.cash/compare/1.0.1...1.0.1-472\r\n**This release contains changes from the following pull requests:**\r\n- [ ] https://github.com/Expensify/Expensify.cash/pull/21\r\n- [x] https://github.com/Expensify/Expensify.cash/pull/22\r\n- [ ] https://github.com/Expensify/Expensify.cash/pull/23\r\n\r\n**Deploy Blockers:**\r\n- [ ] https://github.com/Expensify/Expensify.cash/issues/1\r\n- [x] https://github.com/Expensify/Expensify.cash/issues/2\r\n- [ ] https://github.com/Expensify/Expensify.cash/pull/1234\r\n',
+            };
+
+            const octokit = new Octokit();
+            octokit.repos.listTags = jest.fn().mockResolvedValue({data: [
+                {name: '1.0.1'},
+                {name: '1.0.1-472'},
```

Comment:
Should we chop the third digit from this BUILD just because we are heading that way?

```suggestion
                {name: '1.0.1-42'},
```

Reply:
Good point. We'll need to do that _everywhere_ 😅

Reply:
Updated this to use 2 digits (in this test only for now)